spec/models/work_version_state_machine_spec.rb

```diff
@@ -289,9 +289,15 @@
     end
 
     context 'when a deposit with globus' do
-      let(:work_version) { build(:work_version, :depositing, work:, upload_type: 'globus') }
+      let(:work_version) do
+        build(:work_version, :depositing, work:, upload_type: 'browser', globus_endpoint: '/some/globus/url')
+      end
       let(:collection) { create(:collection) }
```

Comment:
this mimics what an actual globus work version deposit looks like when deposited (i.e. it has upload_type of 'browser' and has an endpoint set